bin/pre-commit-hook.js

```diff
@@ -79,3 +80,32 @@ if ( phpLintResult && phpLintResult.status ) {
 	);
 	process.exit( 1 );
 }
+
+let phpcbfResult, phpcsResult;
+if ( phpcsFiles.length > 0 ) {
+	phpcbfResult = spawnSync( 'vendor/bin/phpcbf', [...phpcsFiles], {
```

Comment:
What standard does it follow here? Should we provide a standard or does it pull from `.phpcs.xml.dist`?

Reply:
It'll pull from .phpcs.xml.dist. Thinking out loud, I haven't tried this from committing not in the jetpack root folder, that might trip things up.

Reply:
Confirmed that trying to commit from a subdir works, no issue.